types/global.d.ts

```diff
@@ -224,13 +228,32 @@ declare class Chrome {
   runtime: Runtime;
 }
 
-type SentryObject = Sentry;
+type SentryObject = Sentry & {
+  getMetaMetricsEnabled: () => Promise<boolean>;
+};
 
 type HttpProvider = {
   host: string;
   timeout: number;
 };
 
+type StateHooks = {
+  getCleanAppState?: () => Promise<any>;
+  getLogs?: () => any[];
+  getMostRecentPersistedState?: () => any;
+  getPersistedState: () => Promise<any>;
+  getSentryAppState?: () => any;
```

Comment:
can we return the shape of state?

Reply:
Unfortunately, not easily, we could add additional and more specific typings in future, but I added the minimum so I could reference these globals from the developer options.